yb-voyager/cmd/importData.go

```diff
@@ -982,8 +976,11 @@ func executeSqlStmtWithRetries(conn **pgx.Conn, sqlInfo sqlInfo, objType string)
 			time.Sleep(time.Second * 5)
 			log.Infof("RETRYING DDL: %q", sqlInfo.stmt)
 			continue
-		} else if strings.Contains(err.Error(), "already exists") && target.IgnoreIfExists {
-			err = nil
+		} else if strings.Contains(err.Error(), "already exists") {
+			fmt.Printf("XXX already exists error: %q\n", sqlInfo.stmt)
+			if target.IgnoreIfExists || strings.EqualFold(strings.Trim(sqlInfo.stmt, " \n"), "CREATE SCHEMA public;") {
```

Comment:
can we comment here why/when `CREATE SCHEMA public` comes and required to be removed.

Reply:
Done.